packages/core/src/app/shipping/Shipping.tsx

```diff
@@ -244,6 +246,7 @@ class Shipping extends Component<ShippingProps & WithCheckoutShippingProps, Ship
             !isEqualAddress(updatedShippingAddress, billingAddress) &&
             !hasRemoteBilling
         ) {
+            console.log('updated shipping address value', updatedShippingAddress);
```

Comment:
if `billingSameAsShipping` and other condition met, it will use shipping address to update billing address